packages/integrations/package.json

```diff
@@ -151,6 +156,7 @@
     "drauu": "^0.3.2",
     "focus-trap": "^7.0.0",
     "fuse.js": "^6.6.2",
+    "idb-keyval": "^6.2.0",
```

Comment:
It also requires to be added to peer deps

Reply:
added to peer deps

Reply:
Regarding the first comment, thank you for your patience as I looked into this.
I looked through `useLocalStorage` and `useSessionStorage` to understand how they reuse `useStorage`. I believe you are suggesting that we could use indexDB storage in place of `window?.localStorage`, the problem I see is that `useStorage` converts all values to strings, and a big reason our team wanted to use indexDB is so we can store values other than just strings. 
I also took a closer look at `StorageLike`, its get and set functions get and return strings, so I am not sure that using indexDB in this way would provide any gains to using `localStorage` as is.
Please let me know if I am misunderstanding something.